Source/Meadow.Foundation.Core/Sensors/Buttons/PushButton.cs

```diff
@@ -134,24 +128,23 @@ public PushButton(IDigitalInputPort inputPort)
 
             LongClickedThreshold = DefaultLongPressThreshold;
 
-            if (DigitalIn.InterruptMode == InterruptMode.EdgeBoth)
+            if (DigitalIn.InterruptMode != InterruptMode.None)
             {
                 DigitalIn.Changed += DigitalInChanged;
             }
             else
-            {   //if we don't have full interrupts, fall back to polling
-                //Re set the resistor mode if Interrupt mode is none - ToDo - known issue
-                if ((DigitalIn.InterruptMode == InterruptMode.None && resistorMode == ResistorMode.InternalPullUp) ||
-                    (DigitalIn.InterruptMode == InterruptMode.None && resistorMode == ResistorMode.InternalPullDown))
-                { DigitalIn.Resistor = resistorMode; }
-
+            {
+                //ToDo remove resistor mode hack for RC2
+                Console.WriteLine($"Resistor mode hack {resistorMode}");
```

Comment:
Will we want this console output when we release?

Reply:
haha nope ... deleted :)